src/repo-intel/gen_techdata.py

```
@@ -0,0 +1,264 @@
+    ["Dockerfile", "Docker", "Tools"],
+    ["docker-compose.yml", "Docker Compose", "Tools"],
+    ["docker-compose.yaml", "Docker Compose", "Tools"],
+    ["compose.yml", "Docker Compose", "Tools"],
+    ["compose.yaml", "Docker Compose", "Tools"],
+    ["Makefile", "Make", "Tools"],
+    ["GNUmakefile", "Make", "Tools"],
+]
+
+
+def fetch(url):
+    req = urllib.request.Request(url, headers={"User-Agent": "repo-intel-gen"})
```

Ruff (0.15.13)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)